### DISCUSSION TITLE
microphone cord and sound very faint

### DISCUSSION THREAD

**Discussion by olup** [nickpourazima/rotary-phone-audio-guestbook#37] [Q&A] [answered]
Hello ! Thanks for this project. I discovered it while trying to build something similar. Thanks for the mic replacement recommendation, i'll follow this route. Isn't it a problem that the handset cord is not screened ? Isn't the sound very bad, then ?

Another question - while trying my usb sound card (plugged in my macbook to try things out without the pi) and having the speaker of the handset plugged into it through a severed jack cord, I get very, very faint sound. I can hear it, but almost not, and that's at the macbook's max level. Do you believe there is something wrong with my unit, or is there some software trick on the pi to push the volume further ? It seems like an amp would be needed. 

Thanks anyway, great work.

**Comment by nickpourazima** [comment, marked as answer]
Hi @olup, apologies for the delayed response as this slipped past me. Can you enter: `alsamixer` in your terminal and boost the gain that way? 

Also, not clear what you mean by the cord not being screened? Are you talking about cable shielding? In that case it's always a good idea, especially if you're noticing 60Hz him when it's plugged into the wall.
